reimbursement.md

```diff
@@ -524,6 +530,7 @@ These errors might be returned in the [request](https://fidel.uk/docs/reimbursem
 fileName:reimbursement-request.json
 {
     "error": {
+        "token": "6c01f956-1f0f-413f-a5db-d1fc8a59ef92",
```

Comment:
We don't want a token here. We actually don't even have a token at this stage

Reply:
Actually I am not sure about this one. I am not sure if this error is returned before the reimbursement is created or not. But if we should add something here, it definitely feels weird to be a token and not an id (same old discussion). Let's discuss this with the team